modules/nw-mutual-tls-auth.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * ingress/ingress-operator.adoc
+
+[id=nw-mutual-tls-auth_{context}]
+= Configuring Mutual TLS Authentication
+
+You can configure the Ingress Controller to enable mutual TLS (mTLS) authentication by setting a `spec.clientTLS` value. The `clientTLS` value configures the Ingress Controller to verify client certificates. This configuration includes setting a `clientCA` value, which is a reference to a config map. The config map contains the PEM-encoded CA certificate bundle that is used to verify a client's certificate. Optionally, you can configure a list of certificate subject filters.
+
+If the `clientCA` value specifies an X509v3 certificate revocation list (CRL) distribution point, the Ingress Operator downloads the CRL and configures the Ingress Controller to acknowledge it. Requests that do not provide valid certificates are rejected.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+Configure Mutual TLS authentication using the `spec.ClientTLS` value for the Ingress Controller.
+
+. Create a ConfigMap that is in the `openshift-config` namespace:
```

Comment:
Please replace ConfigMap with "config map" because it is a [general reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-general-references).